ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/aaa.yml

```diff
@@ -60,10 +60,21 @@ aaa_authorization:
 ## AAA Accounting
 aaa_accounting:
   exec:
+    console:
+      type: start-stop
+      group: TACACS
     default:
       type: start-stop
       group: TACACS
   commands:
+    commands_console:
```

Comment:
I would move change this to `console` since you already nested it under `commands` keys. Same for `default`.

Same for `eos_cli_config_gen`